### PR TITLE
Add /zh-CN/docs/Web/HTTP/Headers/Max-Forwards markdown

### PR DESCRIPTION
**Pull request by sundonghui**

### CODE REVIEW

**Review by yin1999 — CHANGES_REQUESTED** [review]
Hi @HDS3991, we **just** translate `en-US` document. Please read the [translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md).

And translate this document <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Max-Forwards>

**Comment by HDS3991** [comment]
> Hi @HDS3991, we **just** translate `en-US` document. Please read the [translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md).
> 
> And translate this document https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Max-Forwards

Ok, I will translate now.

File: files/zh-cn/web/http/headers/max-forwards/index.md [hunk 1]
```diff
@@ -1,18 +1,40 @@
 ---
 title: Max-Forwards
 slug: Web/HTTP/Headers/Max-Forwards
+tags:
+  - HTTP
+  - HTTP Header
+  - Reference
+  - Request header
+spec-urls: https://httpwg.org/specs/rfc7231.html#header.max-forwards
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/max-forwards/index.md:9]: See the translation guide, we just do not keep this part in translated-content

```suggestion
```

File: files/zh-cn/web/http/headers/max-forwards/index.md [hunk 2]
```diff
@@ -1,18 +1,40 @@
...
+  </tbody>
+</table>
 
 ## 语法
-```plain
-Max-Forwards: <interger>
-```
 
+```http
+Max-Forwards: <integer>
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/max-forwards/index.md:33]: wrong here

File: files/zh-cn/web/http/headers/max-forwards/index.md [hunk 3]
```diff
@@ -22,8 +44,8 @@ Max-Forwards: 10
...
-{{Compat}}
+此功能不是针对浏览器实现的。
+
+## 参见
 
-## 相关内容
-- [Trace](/zh-CN/docs/Web/HTTP/Methods/TRACE)
-- [Options](/zh-CN/docs/Web/HTTP/Methods/OPTIONS)
\ No newline at end of file
+- The HTTP [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 方法
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/max-forwards/index.md:51]: we should remove `The` here:

```suggestion
- HTTP [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 方法
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/http/headers/max-forwards/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: Max-Forwards
+slug: Web/HTTP/Headers/Max-Forwards
+---
+{{HTTPSidebar}}
+
+The **`Max-Forwards`** 请求头被用于限制 [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 方法可经过的服务器（通常指代理服务器）数目。 该字段以十进制整数形式指定可经过的服务器最大数目。服务器在进行下一个 [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 请求之前，`Max-Forwards` 的值将减 1 并重新赋值，直到服务器接收到 `Max-Forwards` 的值为 0 的请求时，则不再进行转发，而是直接返回一个 `200 OK` 的响应。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/max-forwards/index.md:7]:
```suggestion
**`Max-Forwards`** 请求标头被用于限制 [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 方法可经过的服务器（通常指代理服务器）数目。 它的值是一个整数，指定可经过的服务器最大数目。服务器在转发 `TRACE` 请求之前，将递减 `Max-Forwards` 的值，直到到达目标服务器，或服务器接收到 `Max-Forwards` 的值为 0 的请求。而后直接返回一个 `200 OK` 的响应（可以包含一些标头）。
```

File: files/zh-cn/web/http/headers/max-forwards/index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: Max-Forwards
+slug: Web/HTTP/Headers/Max-Forwards
+---
+{{HTTPSidebar}}
+
+The **`Max-Forwards`** 请求头被用于限制 [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 方法可经过的服务器（通常指代理服务器）数目。 该字段以十进制整数形式指定可经过的服务器最大数目。服务器在进行下一个 [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 请求之前，`Max-Forwards` 的值将减 1 并重新赋值，直到服务器接收到 `Max-Forwards` 的值为 0 的请求时，则不再进行转发，而是直接返回一个 `200 OK` 的响应。 
+
+如果 [`TRACE`](/zh-CN/docs/Web/HTTP/Methods/TRACE) 请求中没有 `Max-Forwards` 请求头，就可以认为，不限制可经过的服务器最大数目。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/max-forwards/index.md:9]:
```suggestion
如果 `TRACE` 请求中没有 `Max-Forwards` 标头，就可以认为，不限制可经过的服务器最大数目。
```

File: files/zh-cn/web/http/headers/max-forwards/index.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">Header type</th>
+      <td>{{Glossary("Request header")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">{{Glossary("Forbidden header name")}}</th>
+      <td>no</td>
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/max-forwards/index.md:19]:
```suggestion
      <th scope="row">标头类型</th>
      <td>{{Glossary("Request header", "请求标头")}}</td>
    </tr>
    <tr>
      <th scope="row">{{Glossary("Forbidden header name", "禁止修改的标头")}}</th>
      <td>否</td>
```

File: files/zh-cn/web/http/headers/max-forwards/index.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+Max-Forwards: 10
+```
+
+## 规范
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+此功能不是针对浏览器实现的。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/max-forwards/index.md:43]:
```suggestion
此特性既不针对浏览器，也不在浏览器中实现。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Pull request merged**